code/game/objects/items/devices/taperecorder.dm

```diff
@@ -216,7 +216,6 @@
 			playsleepseconds = (mytape.timestamp[i + 1] - mytape.timestamp[i]) SECONDS
 		if(playsleepseconds > 1.4 SECONDS)
 			sleep(10)
-			atom_say("Skipping [playsleepseconds / 10] seconds of silence.")
 			playsleepseconds = 1 SECONDS
```

Comment:
Do we still need the rest of this part? Won't this make the playback be longer with weird empty pauses?

Reply:
signifying longer delay between messages with longer pauses is better than signifying it with giant blocks of text.
but I suppose it should either be bigger or not be at all.